addons/account/views/account_payment_method_views.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="view_account_payment_method_tree" model="ir.ui.view">
+            <field name="name">account.payment.method.line.tree</field>
+            <field name="model">account.payment.method.line</field>
+            <field name="arch" type="xml">
+                <tree string="Payment Methods">
+                    <field name="sequence" widget="handle"/>
+                    <field name="available_payment_method_ids" invisible="1"/>
+                    <field name="display_name"/>
+                    <field name="payment_method_id"/>
+                    <field name="payment_account_id" placeholder="Let empty to use the default outstanding account"
+                           optional="show" groups="account.group_account_readonly"/>
+                </tree>
+            </field>
+        </record>
+
+        <record id="view_account_payment_method_form" model="ir.ui.view">
+            <field name="name">account.payment.method.line.form</field>
+            <field name="model">account.payment.method.line</field>
+            <field name="arch" type="xml">
+                <form string="New payment method">
+                    <group>
+                        <field name="available_payment_method_ids" invisible="1"/>
+                        <field name="payment_type" invisible="1"/>
+                        <field name="company_id" invisible="1"/>
+                        <group>
+                            <field name="display_name"/>
+                            <field name="payment_method_id" options="{'no_create': True}"/>
+                        </group>
+                        <group>
+                            <field name="payment_account_id"/>
```

Comment:
I don't understand the purpose of such views.
Btw, plz add `groups="account.group_account_readonly"`